config/environments/development.rb

```diff
@@ -20,7 +20,6 @@
     config.action_controller.perform_caching = true
     config.action_controller.enable_fragment_cache_logging = true
 
-    config.cache_store = :memory_store
```

Comment:
Not sure about the extra cache settings in this block, like whether they are necessary or not. For now, not touching the, but memory_store has to be removed otherwise it would override the setting in application.rb.